src/factors/Point2D.jl

```diff
@@ -7,18 +7,21 @@ Direction observation information of a `Point2` variable.
 mutable struct PriorPoint2{T <: IIF.SamplableBelief} <: IncrementalInference.AbstractPrior
   Z::T
   # empty constructor
-  PriorPoint2{T}() where T = new()
+  # PriorPoint2{T}() where T = new()
   # regular constructor
-  PriorPoint2{T}(dist::T) where {T <: IIF.SamplableBelief} = new{T}(dist)
+  # PriorPoint2{T}(dist::T) where {T <: IIF.SamplableBelief} = new{T}(dist)
 end
+
 # convenience helper and default object
-PriorPoint2(z::T=MvNormal(zeros(2),LinearAlgebra.diagm([0.01;0.01]))) where {T <: IIF.SamplableBelief} = PriorPoint2{T}(z)
+PriorPoint2() = PriorPoint2(MvNormal(zeros(2),LinearAlgebra.diagm([0.01;0.01])))
+
+DFG.getManifold(::PriorPoint2) = TranslationGroup(2)
 
 function getSample(cfo::CalcFactor{<:PriorPoint2}, N::Int=1)
-  return (rand(cfo.factor.Z, N),)
+  return [rand(cfo.factor.Z) for _=1:N]
```

Comment:
Must still be in a tuple
```suggestion
  return ([rand(cfo.factor.Z) for _=1:N], )
```